tutorials/plot_dics.py

```diff
@@ -0,0 +1,282 @@
+# -*- coding: utf-8 -*-
+"""
+DICS for power mapping
+======================
+
+In this tutorial, we're going to simulate two signals originating from two
+locations on the cortex. These signals will be sine waves, so we'll be looking
+at oscillatory activity (as opposed to evoked activity).
+
+We'll be using dynamic imaging of coherent sources (DICS) [1]_ to map out
+spectral power along the cortex. Let's see if we can find our two simulated
+sources.
+"""
+# Author: Marijn van Vliet <w.m.vanvliet@gmail.com>
+#
+# License: BSD (3-clause)
+
+###############################################################################
+# Setup
+# -----
+# We first import the required packages to run this tutorial and define a list
+# of filenames for various things we'll be using.
+import os.path as op
+import numpy as np
+from scipy.signal import welch, coherence
+from mayavi import mlab
+from matplotlib import pyplot as plt
+
+import mne
+from mne.simulation import simulate_raw
+from mne.datasets import sample
+from mne.minimum_norm import make_inverse_operator, apply_inverse
+from mne.time_frequency import csd_epochs
```

Comment:
this is deprecated, and there is an error on this file on CircleCI

https://circleci.com/gh/mne-tools/mne-python/7531?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link